www/source/docs/blueprints.md

```diff
@@ -0,0 +1,58 @@
+# Blueprints
+Blueprints are default implementations of the build and runtime phases for your application. They also provide some default health check hooks where appropriate to ensure your application functioning reliably. Customized Blueprints can be created to facilitate reusability of common patterns in your organization for developing, building, and running your applications.
+
+## Getting Started
+### New Projects
+Simply run `hab plan init` to leverage blueprints. Habitat will look in your application code for known project types and the appropriate `pkg_blueprint`.
+
+If you would like to specify the desired blueprint to use, you can run `hab plan init java8-maven-tomcat`
+
+### Existing Plans
+
+To begin using Blueprints, you will need to add the appropriate `pkg_blueprints` 
+
+```bash
+pkg_name="my_java_app"
+....
+pkg_blueprints=("core/blueprints/java8-maven-tomcat")
+               ^     ^^^^^^^^^^^                    ^
+               |______|_______|_____________________|
+                 Arrays and additional namespace TBD -- just random thoughts at the moment.
+```
+* add `auto` package to `build_deps`
+* set toolchain variable
```

Comment:
In that case, it might make sense to either make a slightly longer prefix (like `pkg_blueprint_`) or go for an alternate `blueprint_` prefix. That would keep the build program free to still control the `pkg_` namespace for any future work we might need and provide Blueprint authors a safer namespace that won't be collided with in the future. It should also be clearer to a Plan author which are core metadata (i.e. "where do I look for the `pkg_name` docs?") and any Blueprint metadata (i.e. "where do I look for the `blueprint_node_build_cmd` docs?")